modules/apps/document-library/document-library-item-selector-web/src/main/resources/META-INF/resources/documents.jsp

```diff
@@ -29,7 +29,7 @@ DLItemSelectorViewDisplayContext dlItemSelectorViewDisplayContext = (DLItemSelec
 	portletURL="<%= dlItemSelectorViewDisplayContext.getPortletURL(liferayPortletResponse) %>"
 	repositoryEntries="<%= dlItemSelectorViewDisplayContext.getRepositoryEntries() %>"
 	repositoryEntriesCount="<%= dlItemSelectorViewDisplayContext.getRepositoryEntriesCount() %>"
-	showBreadcrumb="<%= true %>"
+	showBreadcrumb="<%= dlItemSelectorViewDisplayContext.isShowBreadcrumb() %>"
```

Comment:
I told ya! 😄 How about we create a pull and send to the Product Team in charge of if to get its feedback?

Reply:
The Product Team in charge of it is `liferay-lima`. Besides, we can ask its opinion regarding instead of hide we just disable the breadcrumb navigation, what do you think, @carolmariaabb?